corehq/apps/sms/forms.py

```diff
@@ -381,6 +390,14 @@ def section_registration(self):
                     "[project] worker [username]' (where [project] is your "
                     " project space and [username] is an optional username)"
                     ", and the system will add them as a mobile worker."),
+                data_bind="value: sms_mobile_worker_registration_enabled",
+            ),
+            crispy.Div(
+                hqcrispy.FieldWithHelpBubble(
+                    "sms_worker_registration_alert_emails",
+                    help_bubble_text=_("Email these people when new users sign register through SMS"),
```

Comment:
Are you using "register" as a noun, like "sign the register", or did you mean to change "sign up" to "register"?

```suggestion
                    help_bubble_text=_("Email these people when new users register through SMS"),
```

Reply:
🦅 👁️